src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -170,12 +172,13 @@ async function queryFlameGraph(
   }
 
   // profiling-stacktraces is configured with 16 shards
-  const nQueries = 8;
+  const nQueries = parallel ? 8 : 1;
+  parallel = !parallel;
```

Comment:
I am not sure I understand what we're doing here: isn't `parallel` going to be always `true` when we execute this?

Reply:
No, this is how you toggle a boolean value :-)

So this is already been deployed for testing. We want to keep it for a while until we are absolutely sure that parallel requests are good or not.